src/software/ai/navigator/obstacle/obstacle_factory.cpp

```diff
@@ -2,17 +2,15 @@
 
 ObstacleFactory::ObstacleFactory(std::shared_ptr<const ObstacleFactoryConfig> config)
     : config(config),
-      shape_expansion_amount(config->RobotObstacleInflationFactor()->value() *
-                             ROBOT_MAX_RADIUS_METERS)
+      obstacle_expansion_amount(config->RobotObstacleInflationFactor()->value() *
```

Comment:
this value is stored once in the constructor from a dynamic param, it won't update on change after this

Reply:
ObstacleFactory is re-constructed constantly by the Navigator